### ISSUE TITLE
How to create GTA-V LIDAR data?

### ISSUE THREAD

**Issue by gyubeomim** [oscarmcnulty/gta-3d-dataset#1]
Hello :-) 
thank you for sharing such a nice dataset in advance.

I'm researching autonomous vehicle and I need big data to simulate my study.
then It'd be nice to use GTA-V dataset but there was no LIDAR Point Cloud data so far.

and you are sharing GTA-V 3D dataset and I just wondering how did you extract Point Cloud data from GTA-V

**Comment by oscarmcnulty** [comment]
I generated the data for my undergrad thesis which I've uploaded: https://github.com/oscarmcnulty/gta-3d-dataset/blob/master/3D-object-localisation-with-cnns.pdf
See chapter 3 for the conceptual explanation of how to generate the pseudo-LIDAR data. I haven't got around to cleaning up and open sourcing the code I used. I used an AWS EC2 GPU instance with Windows Server 2012 and used a combo of steam streaming to run GTA and windows rdp to run code that hooks into GTA and directX APIs.

Have a look at https://github.com/umautobots/GTAVisionExport for how to do hook into the intermediate directX stages to get things like depth map.
